### PR TITLE
Kubernetes Objects: Updated links to docs resources

### PR DESCRIPTION
**Pull request by nkuba**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7207)

### CODE REVIEW

File: docs/concepts/overview/working-with-objects/kubernetes-objects.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ This page explains how Kubernetes objects are represented in the Kubernetes API,
...
 
-To work with Kubernetes objects--whether to create, modify, or delete them--you'll need to use the [Kubernetes API](https://git.k8s.io/community/contributors/devel/api-conventions.md). When you use the `kubectl` command-line interface, for example, the CLI makes the necessary Kubernetes API calls for you; you can also use the Kubernetes API directly in your own programs. Kubernetes currently provides a `golang` [client library](https://github.com/kubernetes/client-go) for this purpose, and other language libraries (such as [Python](https://github.com/kubernetes-incubator/client-python)) are being developed.
+To work with Kubernetes objects--whether to create, modify, or delete them--you'll need to use the [Kubernetes API](/docs/concepts/overview/kubernetes-api/). When you use the `kubectl` command-line interface, for example, the CLI makes the necessary Kubernetes API calls for you. You can also use the Kubernetes API directly in your own programs using one of [Client Libraries](/docs/reference/client-libraries/.
```
Comment (tengqm) [inline docs/concepts/overview/working-with-objects/kubernetes-objects.md]: The second link now miss a right parenthesis.

File: docs/concepts/overview/working-with-objects/kubernetes-objects.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ In the `.yaml` file for the Kubernetes object you want to create, you'll need to
 * `kind` - What kind of object you want to create
 * `metadata` - Data that helps uniquely identify the object, including a `name` string, UID, and optional `namespace`
 
-You'll also need to provide the object `spec` field. The precise format of the object `spec` is different for every Kubernetes object, and contains nested fields specific to that object. The [Kubernetes API reference](/docs/concepts/overview/kubernetes-api/) can help you find the spec format for all of the objects you can create using Kubernetes.
+You'll also need to provide the object `spec` field. The precise format of the object `spec` is different for every Kubernetes object, and contains nested fields specific to that object. The [Kubernetes API Reference](/docs/reference/) can help you find the spec format for all of the objects you can create using Kubernetes.
```
Comment (tengqm) [inline docs/concepts/overview/working-with-objects/kubernetes-objects.md]: The link should extend to the full link of the api.

**Comment by tengqm** [comment]
@nkuba Signing the CLA is a must for the PR to be merged.

File: docs/concepts/overview/working-with-objects/kubernetes-objects.md (hunk 1 above)
Comment (nkuba) [reply docs/concepts/overview/working-with-objects/kubernetes-objects.md]: Thanks for catching this! Done

File: docs/concepts/overview/working-with-objects/kubernetes-objects.md (hunk 2 above)
Comment (nkuba) [reply docs/concepts/overview/working-with-objects/kubernetes-objects.md]: Done

**Comment by nkuba** [comment]
@tengqm CLA status has updated to signed.

**Pull request merged**